index.html

```diff
@@ -4,34 +4,65 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+    <title>Project 1</title>
+    <style>
+        h1,
+        h3 {
+            font-family: Arial, Helvetica, sans-serif;
+            color: #f4e1d2;
+        }
+
+        h1 {
+            text-decoration: underline;
+        }
+
+        h3 {
+            font-style: italic;
+        }
+
+        p.hi {
+            font-size: 15px;
+        }
+    </style>
 </head>
 
 <body>
-    on paper
-    by Jacqueline Woodson
+    <div class="container" id="container">
+        <div id="container3">
+            <h1 id="text2">On Paper</h1>
+            <h3 id="text3">by Jacqueline Woodson</h3>
+            <div id="container2">
+                <pre style="word-wrap:break-word; word-wrap:white-space; word-wrap:pre-wrap;color:#f4e1d2" id="hi"
```

Comment:
I like the use of `pre` here -- totally valid. However, 2 things to address:

1. All your styles should be in your `index.css` file -- having a mix of external, internal and inline CSS is unnecessary for this project.
2. You've added an ID and class name to this element -- neither of which are used in your CSS. In general, default to using classnames over IDs and only add them to your markup as necessary.